frontend/packages/@depmap/data-explorer-2/src/utils/extend-react-select.tsx

```diff
@@ -189,10 +193,87 @@ export default function extendReactSelect(
                 }),
                 ...props.styles,
               }}
+              onFocus={(e) => {
+                if (!value?.label) {
+                  return;
+                }
+
+                // HACK: Make it appear as if the cursor is at the end of the
```

Comment:
wow, crazy... I certainly don't know of a better alternative so I'll go with, if it looks right, it is right. Interacting with it will teach us whether there are any additional edge cases that we need to handle or not.

I appreciate the comment explaining how this works.